Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "grcov"
 version = "0.8.24"
 authors = ["Marco Castelluccio <mcastelluccio@mozilla.com>"]
 description = "Rust tool to collect and aggregate code coverage data for multiple source files"
-license = "MPL-2.0"
+license = "MPL-2.0 AND MIT" # MIT is used for the bundled Bulma CSS
```

Comment:
We can release the software as MPL-2.0 even if it contains MIT code, so we don't need to modify this.

Reply:
Reverted.